ScaleformUI_Lua/src/scaleforms/scaleform.lua

```diff
@@ -73,7 +73,11 @@ function Scaleform:CallFunction(theFunction, returndata, ...)
 
                     if labelType == "table" then
```

Comment:
Again, very likely to scrap this, its not the best solution, late night thinking is bad.